map/src/login/DownloadBackupDialog.jsx

```diff
@@ -68,6 +69,13 @@ export default function DownloadBackupDialog({ openDownloadBackupDialog, setOpen
             let groups = getCategoryGroups();
             groups = prepareFileGroups(groups);
             setCategories(groups);
+            let res = [];
```

Comment:
use const (with const you can modify const [] and {} but can't reassign it to another arr/obj)